lib/pages/LandingPage.dart

```diff
@@ -151,39 +150,6 @@ class _LandingPageState extends State<LandingPage> {
     return Container();
   }
 
-  Widget reloadPageButton(double tradingTextSize) {
-    return Column(
-      crossAxisAlignment: CrossAxisAlignment.center,
-      children: [
-        Container(
-          child: Text(
-            'Could not load Athlete\'s information',
-            style: TextStyle(
-                fontSize: 14,
-                color: Colors.red,
-                decoration: TextDecoration.none),
-          ),
-        ),
-        Container(
-          padding: EdgeInsets.all(15),
-          decoration: isWeb
-              ? boxDecoration(Colors.transparent, 100, 1, Colors.red[400]!)
-              : boxDecoration(Colors.red[300]!.withOpacity(0.15), 100, 1,
-                  Colors.transparent),
-          child: TextButton(
-            onPressed: () {
-              setState(() {});
-            },
-            child: Text(
-              "Reload the App",
-              style: textStyle(Colors.red[400]!, tradingTextSize, true, false),
-            ),
-          ),
-        ),
-      ],
-    );
-  }
-
   Widget startTradingButton(double tradingTextSize) {
```

Comment:
Please stop using functions to build widgets, unless it's a `builder`.

Reply:
Done, I created custom widgets and separated them into different files